otx/algorithms/common/adapters/mmcv/hooks/recording_forward_hook.py

```diff
@@ -39,13 +40,15 @@ class BaseRecordingForwardHook(ABC):
         module (torch.nn.Module): The PyTorch module to be registered in forward pass
         fpn_idx (int, optional): The layer index to be processed if the model is a FPN.
                                   Defaults to 0 which uses the largest feature map from FPN.
+        normalize (bool, optional): Whether to normalize the resulting saliency maps.
```

Comment:
```suggestion
        normalize (bool): Whether to normalize the resulting saliency maps.
```
It shouldn't get `None` right?